### DISCUSSION TITLE
Stop and resume the spider where it stopped

### DISCUSSION THREAD

**Discussion by shellking4** [elixir-crawly/crawly#234] [General]
Can we stop and resume the spider at a given point ? How is it done ?

**Comment by Ziinc** [comment]
You can call `Crawly.Engine.stop_spider(MySpider)` from anywhere in your project code.

**Comment by Ziinc** [comment]
please use discussions for such question

**Comment by mvkvc** [comment]
Would it be possible if writing to CSV to continue later with the same file and only add new items that are not already there?

**Reply by oltarasenko** [reply]
Hey. Crawly does not support it right now. We may build it as a feature in the future. However, this approach might have some limitations (I am thinking of the cases when the website is being updated in between crawls) compared to daily crawls.

**Reply by mvkvc** [reply]
Thank you! Good to understand.
